src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java

```diff
@@ -207,6 +207,16 @@ public class Mining {
 	@ConfigAccordionId(id = 2)
 	public int forgeDisplayEnabledLocations = 0;
 
+	@Expose
+	@ConfigOption(
+		name = "Forge Tab",
+		desc = "Only show the forge display when tab list is open\n" +
+			"\u00A7cThis only works outside of Dwarven Caves!"
```

Comment:
Not sure if Dwarven Caves is right

Reply:
Dwaven Caves is used by hypixel to descripe both Mines and Crystal Hollows 

See usage of Royal Pigeon outside of Dwarven Caves
![image](https://user-images.githubusercontent.com/69345714/188318726-12de9ba5-ab53-477e-9b11-f360d2991c20.png)